package.json

```diff
@@ -24,13 +25,15 @@
     "sax": "^1.1.3"
   },
   "devDependencies": {
+    "@types/node": "^7.0.12",
     "assert-diff": "^1.0.1",
     "istanbul": "*",
     "mocha": "*",
     "muk-prop": "^1.0.0",
     "nock": "*",
     "sinon": "^2.0.0",
-    "stream-equal": "~1.0.0"
+    "stream-equal": "~1.0.0",
+    "typescript": "^2.2.2"
```

Comment:
This could be debatable I agree, but since those are devDependencies, and they are for working on `this` specific module, you could also argue mocha and instanbul are not needed, since those are test engines used in many popular projects already.
But you obviously need those 2 to run tests whether through travis or offline when you add new features.
If you want me to remove it, its fine though.